discord/guild.py

```diff
@@ -1934,6 +1954,17 @@ async def edit(
         reason: Optional[:class:`str`]
             The reason for editing this guild. Shows up on the audit log.
 
+        raid_alerts_disabled: :class:`bool`
+            Whether the alerts for raid protection should be disabled for the guild.
+
+            .. versionadded:: 2.3
+
+        safety_alerts_channel: :class:`TextChannel`
```

Comment:
This should be `Optional`.

Reply:
It does not have an effect if `safety_alerts_channel` is set to `None`. I thought would be better to prevent confusion since it's not possible on Discord's site either.